.rubocop.yml

```diff
@@ -51,6 +51,10 @@ Layout/ClassStructure:
 Layout/IndentHeredoc:
   EnforcedStyle: powerpack
 
+# Trailing white space is meaningful in code examples
+Layout/TrailingWhitespace:
+  AllowInHeredoc: true
```

Comment:
I disagree with this decision. See e.g. #6190 where (at the time of writing) trailing whitespace is marked for removal by accident. I think that if trailing whitespace is important, it should be written in a string, e.g.
```rb
"string   \n"
```